snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -32,6 +32,27 @@ func snapshot(_ name: String, waitForLoadingIndicator: Bool = true) {
     Snapshot.snapshot(name, waitForLoadingIndicator: waitForLoadingIndicator)
 }
 
+func startRecording(name: String) {
+    sendCommand(commnad: "startRecording",args: "name=\(name)")
+}
+
+func stopRecording() {
+    sendCommand(commnad: "stopRecording")
+}
+
+func sendCommand(commnad: String,args: String = "") {
```

Comment:
There is a typo in the attribute name: `commnad`

Reply:
solved